app/models/intake.rb

```diff
@@ -61,28 +61,8 @@
 #  email_address_verified_at                            :datetime
 #  email_domain                                         :string
 #  email_notification_opt_in                            :integer          default("unfilled"), not null
-#  encrypted_bank_account_number                        :string
-#  encrypted_bank_account_number_iv                     :string
-#  encrypted_bank_name                                  :string
-#  encrypted_bank_name_iv                               :string
-#  encrypted_bank_routing_number                        :string
-#  encrypted_bank_routing_number_iv                     :string
-#  encrypted_primary_ip_pin                             :string
-#  encrypted_primary_ip_pin_iv                          :string
-#  encrypted_primary_last_four_ssn                      :string
-#  encrypted_primary_last_four_ssn_iv                   :string
-#  encrypted_primary_signature_pin                      :string
-#  encrypted_primary_signature_pin_iv                   :string
-#  encrypted_primary_ssn                                :string
-#  encrypted_primary_ssn_iv                             :string
-#  encrypted_spouse_ip_pin                              :string
-#  encrypted_spouse_ip_pin_iv                           :string
 #  encrypted_spouse_last_four_ssn                       :string
```

Comment:
Why are these two,` encrypted_spouse_last_four_ssn and encrypted_spouse_last_four_ssn_iv` , also not deleted?